minvws/nl-kat-coordination
# Edit report recipe

**NOTE:** This PR uses the PR #3695 

--------------------

### Changes
This PR makes it possible for a user to edit the ReportRecipe. 

- We've added a button to the Scheduled Reports page for each scheduled report in the table. 
- This button redirects the user to the OOI edit page, where the ReportRecipe can be edited.
- After editing, the scheduler will be updated as well (to update the "deadline_at" and the "schedule" (schedule == cron-expression)

### Issue link

Closes #3687 

### Demo
https://github.com/user-attachments/assets/986400fe-a78d-4f48-82ff-4a5c96367be0


### QA notes
- Create a report and choose to schedule it
- Go to the "Scheduled" tab on the Reports page
- Open the row in the table
- Click on the edit button
- Try to edit all fields
- Save
- Check if the fields have been changed on the detail page
- Also check if the fields have been changed on the Scheduled Reports page

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
